### PR TITLE
Further corrected documents placed within 120 chars, Ref 73. 

### PR DESCRIPTION
**Pull request by Jigidy**
Ref https://github.com/OpenMS/OpenMS-docs/issues/73.

## Describe the change

Further documents from the "docs" folder have been updated to lay out within 120 character spaces, all documents from additional resources and faqs are now corrected. 
Screenshot 2022-06-22 at 1 15 12 PM

Screenshot 2022-06-22 at 1 15 36 PM

## PR checklist

- [X] I have added description of the change I'm proposing in the OpenMS Documentation.
- [X] I have read and followed [OpenMS Documentation Contributing guidelines](CONTRIBUTING.md).
- [X] I have attached a screenshot of the relevant area after this change.
- [X] `CHANGELOG.md` is updated.
- [X] I have added my name in [CONTRIBUTING.md](CONTRIBUTING.md#openms-documentation-contributors).

### CODE REVIEW

File: docs/additional-resources/developer-guidelines-for-addding-new-dependent-libraries.md [hunk 1]
```diff
@@ -35,27 +35,27 @@ these platforms.
 - on **WindowsOS** this usually means, adding the new library to the Contrib in debug and release variants. In short all
   recent versions of Visual Studio (VS2008 and onwards) must be supported (or support must be added). This encompasses
   - a solution file (which can be either statically present or generated by a meta-system like CMake) is available
-  - The library actually compiles and is linked to the **dynamic** VS-C++ runtime lib (since this is what the OpenMS lib will
-  link to as well - combining static and dynamic links will lead to linker errors or segfaults).
+  - The library actually compiles and is linked to the **dynamic** VS-C++ runtime lib (since this is what the OpenMS lib
+  will link to as well - combining static and dynamic links will lead to linker errors or segfaults).
```
Comment (tapaswenipathak) [inline docs/additional-resources/developer-guidelines-for-addding-new-dependent-libraries.md:41]:
```suggestion
    will link to as well - combining static and dynamic links will lead to linker errors or segfaults).
```

File: docs/additional-resources/developer-guidelines-for-addding-new-dependent-libraries.md [hunk 2]
```diff
@@ -35,27 +35,27 @@ these platforms.
...
-  also be present.
+  those libraries instead of shipping them via the contrib. Additionally, the MacPorts and HomeBrew formulas for 
+  building the libraries can serve as blueprints on how to compile the library in a generic setting inside the contrib 
+  which should also be present.
 
-- on **Linux** since we (among other distributions) feature an OpenMS Debian package which requires that all dependencies
-  of OpenMS are available as Debian package as well, the new library must be available (or made available) as Debian
-  package or linked statically during the OpenMS packaging build.
+- on **Linux** since we (among other distributions) feature an OpenMS Debian package which requires that all 
+dependencies of OpenMS are available as Debian package as well, the new library must be available (or made available) as
```
Comment (tapaswenipathak) [inline docs/additional-resources/developer-guidelines-for-addding-new-dependent-libraries.md:51]:
```suggestion
    dependencies of OpenMS are available as Debian package as well, the new library must be available (or made available) as 
```

File: docs/additional-resources/external-code-using-openms.md [hunk 3]
```diff
@@ -112,7 +112,8 @@ In short:
 - copy the `OpenMS/share/OpenMS` directory to the client machine (e.g `<client/my_dir>/share`) and set the environment
   variable `OPENMS_DATA_PATH` to this directory
 - copy the OpenMS library (`OpenMS.dll` for Windows or `OpenMS.so/.dylib` for Linux/macOS) to `<client/my_dir>/bin`.
-- copy all Qt4 libraries to the client `<client/my_dir>/bin` or on Linux/macOS make sure you have installed the Qt4 package
+- copy all Qt4 libraries to the client `<client/my_dir>/bin` or on Linux/macOS make sure you have installed the Qt4 
+package
```
Comment (tapaswenipathak) [inline docs/additional-resources/external-code-using-openms.md]:
```suggestion
    package
```

File: docs/additional-resources/openms-git-workflow.md [hunk 4]
```diff
@@ -13,7 +13,8 @@ Naming conventions for the following apply:
 
 * A **local repository** is the repository that lies on your hard drive after cloning.
 * A **remote repository** is a repository on a git server such as GitHub.
-* A **fork** is a copy of a repository. Forking a repository allows you to freely experiment with changes without affecting the original project.
+* A **fork** is a copy of a repository. Forking a repository allows you to freely experiment with changes without 
+affecting the original project.
```
Comment (tapaswenipathak) [inline docs/additional-resources/openms-git-workflow.md]:
```suggestion
   affecting the original project.
```

File: docs/additional-resources/write-and-label-github-issues.md [hunk 5]
```diff
@@ -20,5 +20,6 @@ To label an issue:
    - **enhancement**: An enhancement refers to a feature idea to enhance the current OpenMS code. This is a medium
      priority issue.
    - **task**: A task refers to a single piece of work that a developer can undertake. This is a medium priority issue.
-   - **refactoring**: A refactoring issue refers to a suggestion to streamline the code without changing how the code function.
+   - **refactoring**: A refactoring issue refers to a suggestion to streamline the code without changing how the code 
+   function.
```
Comment (tapaswenipathak) [inline docs/additional-resources/write-and-label-github-issues.md]:
```suggestion
       function.
```

File: docs/faqs/contributor-faq.md [hunk 6]
```diff
@@ -14,7 +14,8 @@ The following section provides general information to new contributors.
 * Read the [OpenMS Coding Conventions](https://github.com/OpenMS/OpenMS/wiki/Coding-conventions).
 * Read the [OpenMS Tutorial](../tutorials/user-tutorial.md).
 * Create a GitHub account.
-* Subscribe to the [open-ms-general](https://sourceforge.net/projects/open-ms/lists/open-ms-general) or [contact us](../contact-us.md).
+* Subscribe to the [open-ms-general](https://sourceforge.net/projects/open-ms/lists/open-ms-general) 
+or [contact us](../contact-us.md).
```
Comment (tapaswenipathak) [inline docs/faqs/contributor-faq.md:18]:
```suggestion
  or [contact us](../contact-us.md).
```

File: docs/faqs/contributor-faq.md [hunk 7]
```diff
@@ -77,15 +78,17 @@ using the `-G` option.
...
 5. Add them to git by using the command `git add`.
 
 ### How do I add a new directory to the build system?
 
 1. Create two new `sources.cmake` files (one for `src/<sub-project>/include/OpenMS/MYDIR`, one for
    `src/<sub-project>/source/MYDIR`), using existing `sources.cmake` files as template.
 2. Add the new `sources.cmake` files to `src/<sub-project>/includes.cmake`
-3. If you created a new directory directly under `src/openms/source`, then have a look at `src/tests/class_tests/openms/executables.cmake`.
+3. If you created a new directory directly under `src/openms/source`, then have a look 
+at `src/tests/class_tests/openms/executables.cmake`.
```
Comment (tapaswenipathak) [inline docs/faqs/contributor-faq.md]:
```suggestion
  at `src/tests/class_tests/openms/executables.cmake`.
```

File: docs/faqs/contributor-faq.md [hunk 8]
```diff
@@ -193,7 +197,8 @@ View the following TOPP and TOPPView tutorials:
 To contribute to a bug fix:
 
 1. Submit the bug as a GitHub issue.
-2. Create a feature branch (e.g. `feature/fix_missing_filename_issue_615`) from your (up-to-date) develop branch in your fork of OpenMS.
+2. Create a feature branch (e.g. `feature/fix_missing_filename_issue_615`) from your (up-to-date) 
+develop branch in your fork of OpenMS.
```
Comment (tapaswenipathak) [inline docs/faqs/contributor-faq.md]:
```suggestion
   develop branch in your fork of OpenMS.
```

**Review by tapaswenipathak — CHANGES_REQUESTED** [review]
minor nits, let's make sure alignment in bullet points (lists) are proper, ex:
```
- this is a list sentence
continuing on another line.
```
```
- this is a list sentence
  continuing on another line.
```

i could have resolved the merge conflict for you, but i can't, please resolve any.

Screenshot 2022-06-23 at 10 12 15 AM

**Comment by Jigidy** [comment]
All lines now aligned properly.

**Review by tapaswenipathak — APPROVED** [review]

**Pull request merged**